### ISSUE TITLE
It crashes

### ISSUE THREAD

**Issue by eric-methorst** [kswoll/WootzJs#40]
Trying to run it I get a null reference exception.
In WootzJs.Compiler.Context.UpdateContext(245)
SpecialFunctions = compilation.FindType("System.Runtime.WootzJs.SpecialFunctions"); -> returns null
consequently crashes at
WootzJs.Compiler.RoslynExtensions(428):
var candidates = type.GetMembers(name).OfType<IMethodSymbol>().ToArray(); -> type is null

Also, this is my view in VS2019:
![image](https://user-images.githubusercontent.com/61496061/93751159-efb26e00-fbfc-11ea-87fb-d4a2f9c74238.png)

Is that correct? because when I attempt to load e.g. WootzJs.Injection i get:
 error  : The imported project "D:\MyProjects\Github\WootzJs\build\WootzJs.targets" was not found. Confirm that the expression in the Import declaration "..\build\WootzJs.targets" is correct, and that the file exists on disk.  [Path]\Github\WootzJs\WootzJs.Injection\WootzJs.Injection.csproj

**Comment by kswoll** [comment]
I should update the readme here, since I am not interested in this project any longer, since I believe Blazor has obviated any need for using this framework.  I'm curious, can you see any scenario in which Blazor is not superior to WootzJs?

**Comment by eric-methorst** [comment]
Doesn’t work on IPads, I’ve been told by a colleague who tried it.

**Comment by kswoll** [comment]
Yeah, I remember it not working on iOS a while ago, but I believe it's been fixed.  Definitely where the future lies, so I'd encourage you to retest.  

(The issue I remember that's fixed is this one:  https://github.com/mono/mono/issues/18646#issuecomment-597772473)

**Comment by eric-methorst** [comment]
Additionally, I’ve been told that blazor creates a mandatory window in which the entire application must run, which to us is not an option, a we’re not going to rewrite hundreds of thousands of lines of javascript.
Web assembly might well be the future, but that future is still a bit off: first we need something we can just plug in to a website, e.g. doing just some calculations we want to run both on the server and on the client
